This patch is required in order to build in 64-bit mode on Solaris.

This patch will not be submitted upstream.

--- include/system.h	2015-12-07 00:26:01.000000000 -0800
+++ include/system.h	2016-04-14 14:53:13.263148993 -0700
@@ -165,7 +165,7 @@ E long FDECL(lseek, (int, long, int));
 #endif
 #if defined(POSIX_TYPES) || defined(__TURBOC__)
 #ifndef bsdi
-E int FDECL(write, (int, const void *, unsigned));
+/* E int FDECL(write, (int, const void *, unsigned)); */
 #endif
 #else
 #ifndef __MWERKS__ /* metrowerks defines write via universal headers */
